tmp/canary-monitor.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: canary-metrics
+  labels:
+    app: strimzi
+spec:
+  selector:
+    matchLabels:
+      strimzi.io/kind: Canary
+  namespaceSelector:
+    matchNames:
+      - my-kafka
```

Comment:
if this is an example you should be clear in the description that the namespace should be `my-kafka` otherwise metrics discovery will not work.

Reply:
Yes. I should add a comment. I thought strimzi quick-guide was also using namespace my-kafka. But it is actually using `my-kafka-project`. So, I can change it to `my-kafka-project` to align it to the quick guide add also a comment above to adjust it for the target clusters. How do you think?